solutions/usaco-860.mdx

```diff
@@ -13,6 +13,131 @@ author: Nathan Gong
 
 <LanguageSection>
 
+<CPPSection>
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+int n, k;
+const int MAX_ROWS = 100;
+const int MAX_COLS = 10;
+char board[MAX_ROWS][MAX_COLS];
```

Comment:
oh uh if you changed `visited` to a vector could you do the same for `board` as well?